log.py

```diff
@@ -52,33 +53,139 @@ def format(self, record):
             data = data.encode("utf8")
         return data
 
-       
+class SysLogger():
```

Comment:
It's better to use `object` as the superclass of Python classes that would otherwise have no superclass.